Automattic/jetpack
# Hide My Mailboxes link on P2 sites

Related to https://github.com/Automattic/dotcom-forge/issues/8069

## Proposed changes:
- Hide `My Mailboxes` sidebar link on P2 sites

### Other information:

This fixes a regression from #32992, where the `My Mailboxes` link was renamed from `Inbox`. The `Inbox` link was previously hidden, but the renaming made the function call for removing the link void.


## Jetpack product discussion



https://github.com/Automattic/dotcom-forge/issues/8069 by way of p58i-hxi-p2

## Does this pull request change what data or activity we track or use?



No

## Testing instructions:






* Follow [these instructions](https://github.com/Automattic/jetpack/pull/38232#issuecomment-2213578280) to prep your sandbox for testing this PR
* Sandbox a P2 domain name (e.g., `examplep2.wordpress.com`)
* Navigate to `/wp-admin` on a P2 site
* Ensure that the sidebar doesn't contain a `My Mailboxes` link
